spyder/plugins/completion/plugin.py

```diff
@@ -367,7 +367,7 @@ def after_configuration_update(self, options: List[Union[tuple, str]]):
             if opt in options:
                 opt_value = self.get_conf(opt, section=sec)
                 self.sig_editor_rpc.emit('call_all_editorstacks',
-                                         (method_name, (opt_value,)),
+                                         (method_name, opt_value,),
```

Comment:
@dalthviz, did you test that this change works for all other `editor_method_sec_opts` items?

Reply:
I mean, I say it because perhaps it breaks applying other options listed there.

Reply:
@ccordoba12 no, I was hoping @andfoy could tell me if some setter there needs the tuple but then I will check :+1:

Reply:
This should be a tuple, due to the signature of this method: https://github.com/spyder-ide/spyder/blob/5c8cca33fe2cc012efdffe2c57dcf8a1143f3386/spyder/plugins/editor/plugin.py#L408

Reply:
Since the signal is only used to set options, maybe we could simplify it to consider a single argument?

Reply:
From what I see and tested seems to work. Actually, seems like other options were using the truthy value of the tuple (so even if you uncheck them they didn't disable)

Reply:
Just in case, checking a little bit more, seems like the tuple could be for the `edge_line_columns` however the panel handles non-tuple values too. The type of the values for the setters options are:

* `enable_code_snippets`: bool
* `enable_hover_hints`: bool
* `format_on_save`: bool
* `automatic_completions`: bool
* `completions_hint`: bool
* `completions_hint_after_ms`: int 
* `underline_errors`: bool
* `automatic_completions_after_chars`: int
* `automatic_completions_after_ms`: int
* `pycodestyle/max_line_length`: tuple
* `edge_line`: bool

Reply:
@andfoy I didn't see your message above but I think I now know were the actual issue is :+1:

Reply:
I think there is something strange passing the `args` and `kwargs` currently that is causing that they arrive to the setter methods still as a tuple (instead of being expanded when calling the corresponding setter)